crates/pipeline_manager/migrations/V0__ddl_base.sql

```diff
@@ -1,59 +1,99 @@
 CREATE TABLE IF NOT EXISTS tenant (
```

Comment:
Since we moved to using migrations, this file shouldn't be edited. Instead, add a new migration file `V1__versioning.sql` that incrementally adds the schema changes using CREATE table for the history tables and ALTER table to edit the schema of the V0 tables.

Then add a test that sets the migration to V0 and creates a few objects before calling ProjectDB::connect() (which will update the schema + objects to V1).

Reply:
Yea I thought about it but was kind of hoping not to start versioning the SQL not with this change and instead something smaller which hopefully most stuff will be smaller for a while after this

Reply:
It's actually the ideal kind of patch to try it out as opposed to ones with more minute changes. It should just work out of the box (I can add the V0 -> V1 tests separate from this PR if you'd like).